pkg/users/password-dialogs.js

```diff
@@ -137,11 +138,10 @@ function SetPasswordDialogBody({ state, errors, change }) {
             <>
                 <input hidden disabled value={current_user} />
                 <FormGroup label={_("Old password")}
-                           helperTextInvalid={errors?.password_old}
-                           validated={(errors?.password_old) ? "error" : "default"}
                            fieldId="account-set-password-old">
                     <TextInput className="check-passwords" type="password" id="account-set-password-old"
                                autoComplete="current-password" value={password_old} onChange={value => change("password_old", value)} />
+                    <FormHelper helperTextInvalid={errors?.password_old} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18641-20230417-051207-efb10d39-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)